fixtures/ayra/tdc-triple-burst.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
   "name": "TDC Triple Burst",
   "shortName": "AyraTDCTripleBurst",
-  "categories": ["Color Changer", "Effect"],
+  "categories": ["Pixel Bar", "Color Changer", "Effect"],
```

Comment:
It is questionable whether this fixture really is a "bar". The beams aren't parallel and it's more a multi-beam effect than a pixel effect.

See demo video: https://www.youtube.com/watch?v=Xt2d_Y9eGzs (not included in definition yet)